cfgov/v1/atomic_elements/organisms.py

```diff
@@ -1299,3 +1299,50 @@ class Meta:
         icon = 'image'
         label = 'CCT Data Snapshot'
         template = '_includes/organisms/data_snapshot.html'
+
+
+class ImageMapCoordinates(blocks.StructBlock):
+    left = blocks.FloatBlock(required=True)
+    top = blocks.FloatBlock(required=True)
+    width = blocks.FloatBlock(required=True)
+    height = blocks.FloatBlock(required=True)
+
+
+class ExplainerNote(blocks.StructBlock):
+    coordinates = ImageMapCoordinates(
+        form_classname='coordinates',
+        label='Note image map coordinates',
+        help_text='Enter percentage values to define the area '
+                  'that will be highlighted on the image for this note.')
+    heading = blocks.CharBlock(required=True, label='Note heading')
+    body = blocks.RichTextBlock(required=True, label='Note text')
```

Comment:
If this really needs to be rich text, could the rich text elements be limited, e.g. to just links? Will users need to be able to embed images, etc. here?

Reply:
There are lots of links to internal resources in the existing explainer notes, so I think the rich text field is necessary. That's a great point about other content types, though -- I limited the options for that field to just the basics in f6965b39e1c05ea1e6c3b58b134e1da6333e41f8.